apps/web/pages/index.tsx

```diff
@@ -1,52 +1,17 @@
 import { useRouter } from 'next/router';
-import { useState } from 'react';
+import { useEffect } from 'react';
 
-const DEFAULT_ROOT_COMPONENT = 'andyh.near/widget/ComponentIdTestRoot';
+const DEFAULT_ROOT_COMPONENT = 'andyh.near/widget/LandingPage';
```

Comment:
I'd like to get this injected at build time, what's the ideal way of doing this in Vercel?